src/modules/rlm_json/json.c

```diff
@@ -252,3 +252,105 @@ void fr_json_version_print(void)
 #endif
 }
 
+static json_object *add_vp_to_json_array(TALLOC_CTX *ctx, json_object *arr, VALUE_PAIR *vp) {
```

Comment:
verb last, file first, so

	json_array_add_vp

There is a document on the wiki.  If it's unclear in parts, let me know and i'll add more examples.